doc/tutorials/imgproc/morph_lines_detection/moprh_lines_detection.md

```diff
@@ -0,0 +1,87 @@
+Extract horizontal and vertical lines by using morphological operations {#tutorial_moprh_lines_detection}
+=============
+
+Goal
+----
+
+In this tutorial you will learn how to:
+
+-   Apply two very common morphology operators (i.e. Dilation and Erosion), with the creation of custom kernels, in order to extract straight lines on the horizontal and vertical axes. For this purpose, you will use the following OpenCV functions:
+    -   @ref cv::erode
+    -   @ref cv::dilate
+    -   @ref cv::getStructuringElement
+
+    in an example where your goal will be to extract the music notes from a music sheet.
+
+Theory
+------
+
+### Moprhology Operations
+@brief Morphology is a set of image processing operations that process images based on predefined *structuring elements* known also and as kernels. The value of each pixel in the output image is based on a comparison of the corresponding pixel in the input image with its neighbors. By choosing the size and shape of the kernel, you can construct a morphological operation that is sensitive to specific shapes regarding the input image.
```

Comment:
Shouldn't the `and` be removed in this sentence: `... known also and as kernels`?